StockWatch.Backend/src/main/java/com/henning/oden/java/StockAlert/Backend/services/CustomUserDetailsService.java

```diff
@@ -0,0 +1,38 @@
+package com.henning.oden.java.StockAlert.Backend.services;
+
+import com.henning.oden.java.StockAlert.Backend.entities.SystemUser;
+import com.henning.oden.java.StockAlert.Backend.repos.SystemUserRepository;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+@Service
+public class CustomUserDetailsService implements UserDetailsService {
```

Comment:
Do we need interface?

Reply:
I tried to remove the interface and the next time I ran the application and tried to sign in, I got a StackOverflowError. Nothing else was changed other than remove the interface and override.